src/views/LOG/inventory/editComponent.vue

```diff
@@ -176,6 +176,24 @@ export default {
   },
 
   methods: {
+    logTransfer(id, to) {
```

Comment:
what is this doing?

Reply:
I did not make this change but the arguments are used in a post request to transfer products. {to: to_id,  id, inventory_id}